### ISSUE TITLE
Suggestion: convert existing videos to Tips Of The Day

### ISSUE THREAD

**Issue by aaaaalbert** [pd-externals/tipoftheday-plugin#39]
There are a couple of videos out there which demonstrate cool, useful patching etc. features. They could be cut up and converted to *Tips Of The Day* with (relative) ease if the original authors agree for the material to be used, or otherwise re-done from scratch.

Here is [one example](https://vimeo.com/340437816) I learnt of recently, linking to pure-data/pure-data#575 ("intelligent patching", look at that lovely tabbing feature during making connections!).

Note: I don't suggest that *I* have to be the person to do that, nor that *you, dear reader* should feel obliged to. I'm just pointing out the existence of generally suitable material :-)

**Comment by umlaeute** [comment]
while i love the example you gave (being the author), i think this is too long for a tip-of-the-day.

otoh, the various intelligent-patching videos could be easily split up into multiple short animated GIFs...

> Note: I don't suggest that I have to be the person to do that

which is a pity.
i would welcome such contributions.

**Comment by aaaaalbert** [comment]
> while i love the example you gave (being the author), i think this is too long for a tip-of-the-day.
> 
> the various intelligent-patching videos could be easily split up into multiple short animated GIFs...

Yes, that's what I was thinking too. I could split e.g. the "cycle-connect" section into separate Tips showing tabbing between outlets and inlets.

Oh, wait -- did I just volunteer? :open_mouth: 

-----

In earnest, what do you think of making this issue a (task) list of videos that should be split up and recreated as Tips Of The Day? And whenever someone works on one, they create a new GitHub issue, and check it off the list here.

My problem: I know only this video, and vaguely remember one about "intelligent patching" (probably by you @umlaeute as well?). Furthermore, I'm not sure if task lists can be edited / appended to by people other than the respective author. **Suggestions welcome!**

**Comment by umlaeute** [comment]
sounds good.
